x/virtualgroup/keeper/keeper.go

```diff
@@ -163,12 +165,19 @@ func (k Keeper) DeleteGVG(ctx sdk.Context, primarySp *sptypes.StorageProvider, g
 	}
 
 	if len(gvgFamily.GlobalVirtualGroupIds) == 0 && k.paymentKeeper.IsEmptyNetFlow(ctx, sdk.MustAccAddressFromHex(gvgFamily.VirtualPaymentAddress)) {
-		store.Delete(types.GetGVGFamilyKey(gvg.FamilyId))
-		if err := ctx.EventManager().EmitTypedEvents(&types.EventDeleteGlobalVirtualGroupFamily{
-			Id:          gvgFamily.Id,
-			PrimarySpId: gvgFamily.PrimarySpId,
-		}); err != nil {
-			return err
+		// after Eddystone, the virtual group family can be empty.
+		if !ctx.IsUpgraded(upgradetypes.Eddystone) {
```

Comment:
Can we simplify these codes?
if len(gvgFamily.GlobalVirtualGroupIds) == 0 && k.paymentKeeper.IsEmptyNetFlow(ctx, sdk.MustAccAddressFromHex(gvgFamily.VirtualPaymentAddress)) && !ctx.IsUpgraded(upgradetypes.Eddystone) {
    ....
}

Reply:
fixed.